docs/usage/prune.rst.inc

```diff
@@ -17,7 +17,7 @@ borg prune
     +-----------------------------------------------------------------------------+---------------------------------------------------+-----------------------------------------------------------------------------------------------------------------------------------+
     |                                                                             | ``REPOSITORY``                                    | repository to prune                                                                                                               |
     +-----------------------------------------------------------------------------+---------------------------------------------------+-----------------------------------------------------------------------------------------------------------------------------------+
-    | **optional arguments**                                                                                                                                                                                                                                              |
+    | **options**                                                                                                                                                                                                                                                         |
```

Comment:
you don't need to include the generated file(s) into the PR.

i'll generate them as part of the release process.